### PR TITLE
assets: move to webpack 

### PR DESCRIPTION
**Pull request by rerowep**
changed to webpack

### CODE REVIEW

File: rero_mef/theme/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of RERO MEF.
+# Copyright (C) 2018 RERO.
```
Comment (iGormilhit) [inline rero_mef/theme/__init__.py:4]: Copyright (C) 2020 RERO.

**Review by iGormilhit — COMMENTED** [review]
Commit message approved.

Just a copyright date to update.

Is it on purpose that you have 6 commits in this PR?

File: rero_mef/theme/__init__.py (hunk 1 above)
Comment (rerowep) [reply rero_mef/theme/__init__.py]: After the merge, we can change all the license to the one we want https://github.com/rero/rero-mef/issues/45.

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

**Pull request merged**